### ISSUE TITLE
Need the function to auto unmute.

### ISSUE THREAD

**Issue by Uhtredtb** [yonilevy/automute#9]
Every time, auto mute runs and when I unlock the Mac I need to unmute and turn up the volume, is there any function to auto unmute when Mac is unlock. 

Kind of reverting the function, and run when Mac is wake or login using it.

@yonilevy 
Let me know,
Thanks

**Comment by yonilevy** [comment]
I think what you want is to turn off the "Mac goes to sleep" and "Mac gets locked" mute triggers. The app by default is intentionally muting in those scenarios.

**Comment by Uhtredtb** [comment]
@yonilevy Yes, that is correct. But when the Mac unlock I want the sound back so it won't mute anymore. Something like that because, when I'm not using my Mac all the notification should be silence. Is that possible? please let me know thanks.

**Comment by yonilevy** [comment]
oh I see, makes sense, I'm reopening and will consider adding later!

**Comment by Uhtredtb** [comment]
Thank you, looking forward for an update. @yonilevy
